### DISCUSSION TITLE
Can I Get Github Student Pack as a high school student ?

### DISCUSSION THREAD

**Discussion by Junaid433** [community/community#17789] [GitHub Education]
Hi, I have a question in my mind. I live in Bangladesh and im from a small city named Pancagarh. I study in Bishnu Prashad Government High School, Panchagarh. I am in the 9th class now. Our School doesn't provides students a education mail.

So my question was - am i eligible to get the github students pack ?

**Comment by andhra21231** [comment]
As Your school Have like yourschool@topdomainforschool.in.bd
and you have Student ID card you can apply

**Reply by andhra21231** [reply]
but if you don't have it, try using gmail but im not recommend it

**Comment by juanpflores** [comment]
Hi @Junaid433 
You can also submit a photo id. You can read all about it in the [GitHub Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack).

**Comment by andhra21231** [comment]
You Can Try it, but what i said. I not recommended using personal mail like Gmail, Yahoo Mail, etc

**Comment by suprstarrd** [comment]
By the title alone, being a "high school student" - yes! I'm a high school student and was accepted.

As for the rest of what you said - the best way to apply is with a school-issued email. If your school does not provide a special student email (e.g. @institution.edu, @school.net, or anything of the sort that identifies it as educational), then don't worry! You can still apply. You'll (probably) be asked for "further verification" and here you can upload anything that identifies you as a student. 

I uploaded my library card, but a digital "student dashboard", a student ID, a current transcript, letter on school letterhead, or any other form of documentation that demonstrates your current enrollment works. Make sure the image is clear and easy to read. Languages that aren't English work too but the acceptance process might take longer as it needs to be translated to English for review.

Good luck!
